zetaclient/chains/bitcoin/observer/outbound.go

```diff
@@ -282,29 +278,29 @@ func (ob *Observer) checkTxInclusion(
 	txHash string,
 ) (*btcjson.GetTransactionResult, bool) {
 	// logger fields
-	lf := map[string]any{
-		logs.FieldMethod: "checkTxInclusion",
-		logs.FieldNonce:  cctx.GetCurrentOutboundParam().TssNonce,
-		logs.FieldTx:     txHash,
-	}
+	logger := ob.logger.Outbound.With().
+		Str(logs.FieldMethod, "checkTxInclusion").
+		Uint64(logs.FieldNonce, cctx.GetCurrentOutboundParam().TssNonce).
+		Str(logs.FieldTx, txHash).
+		Logger()
 
 	// fetch tx result
 	hash, txResult, err := ob.rpc.GetTransactionByStr(ctx, txHash)
 	if err != nil {
-		ob.logger.Outbound.Warn().Err(err).Fields(lf).Msg("GetTxResultByHash failed")
+		logger.Warn().Err(err).Msg("GetTxResultByHash failed")
```

Comment:
I think the function name has changed from `GetTxResultByHash` -> `GetTransactionByStr` at some point, so log message needs updating as well.